torch/nn/quantized/modules/linear.py

```diff
@@ -42,26 +42,45 @@ def _weight_bias(self):
     def forward(self, x):
         return x
 
+    # Version 1
+    #   self
+    #   |--- weight : Tensor
+    #   |--- bias : Tensor
+    #
+    # Version 2
+    #   self
+    #   |--- weight : Tensor
+    #   |--- bias : Tensor
+    #   |--- dtype : torch.dtype
+    #
+    # Version 3
+    #   self
+    #   |--- _packed_params : LinearPackedParamsBase
+    #   |--- dtype : torch.dtype
     def _save_to_state_dict(self, destination, prefix, keep_vars):
         super(LinearPackedParams, self)._save_to_state_dict(destination, prefix, keep_vars)
-        (w, b) = self._weight_bias()
-        destination[prefix + 'weight'] = w
-        destination[prefix + 'bias'] = b
         destination[prefix + 'dtype'] = self.dtype
+        destination[prefix + '_packed_params'] = self._weight_bias()
```

Comment:
should this be `destination[prefix + '_packed_params'] = self._packed_params()`?

Reply:
No, this is handling unpacking the weights for serialization

Reply:
why do you unpack here? I think you can do this unpacking even without the torchbind class support right? I thought we want to save the packed param directly here